build-logic-commons/gradle-plugin/src/main/kotlin/gradlebuild.reproducible-archives.gradle.kts

```diff
@@ -16,8 +16,26 @@
 
 // Ensure the archives produced are reproducible
 tasks.withType<AbstractArchiveTask>().configureEach {
-    isPreserveFileTimestamps = false
-    isReproducibleFileOrder = true
+    // TODO: Remove these try-catches once https://github.com/gradle/gradle/pull/31212 is merged, the wrapper for the Gradle build updated, and the properties are removed
```

Comment:
💭 This is a hacky workaround.  IDK if there is a better way to handle usage of properties in the gradle build itself that will be updated.